server/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.8-slim-buster
+
+#RUN groupadd -r scratchy && useradd -m -g scratchy scratchy
+#USER scratchy
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip3 install -r requirements.txt
+
+COPY . .
```

Comment:
copy . . when you copy requirements.txt to reduce the number of image

Reply:
Coping requirements.txt and installing it create a docker layer with python3.8 and scratchy dependency, that means that if requirements.txt doesn't change the layer will be reused and only created once. It's a good practice.